jupyterhubutils/spawner/lsstspawner.py

```diff
@@ -157,18 +163,23 @@ def get_user_namespace(self):
             return "{}-{}".format(defname, self.user.escaped_name)
 
     def start(self):
-        with start_action(action_type="start"):
-            # All we need to do is ensure the namespace and K8s ancillary
-            #  resources before we run the superclass method to spawn a pod,
-            #  so we have the namespace to spawn into, and the service account
-            #  with appropriate roles and rolebindings.
-            self.log.debug("Starting; creating namespace/ancillary objects.")
-            self.log.debug(
-                "User name in start() is '{}'.".format(self._log_name))
-            self.set_user_namespace()  # Set namespace here/in namespace_mgr
-            self.lsst_mgr.ensure_resources()
-            retval = super().start()
-            return retval
+        """Thin wrapper around self._start
+
+        so we can hold onto a reference for the Future
+        start returns, which we can use to terminate
+        .progress()
+        """
+        self._start_future = self._start()
+        return self._start_future
```

Comment:
This looks like a copy of the `start` method in `KubeSpawner`, so can't you just not implement this and inherit it from the parent? That's what it looked like the multinamespace spawner did.

Reply:
I tried that and it didn't call my `_start`.  Not sure what was up with that.